### PR TITLE
Fix IPython dependencies

### PR DESCRIPTION
**Pull request by oroppas**
python-path: Created PKGBUILD
python-pickleshare: Created PKGBUILD
python-simplegeneric: Created PKGBUILD
python-traitlets: Created PKGBUILD
python-ipython: Added python-traitlets, python-simplegeneric, python-pickleshare, python-path, python-ipython_genutils as dependency

### CODE REVIEW

**Comment by oroppas** [comment]
Fixes #937.

File: mingw-w64-python-traitlets/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    nosetests-3.5 test_traitlets.py
+}
+
+package_python3-traitlets() {
+  depends=(
+    "${MINGW_PACKAGE_PREFIX}-python3>=3.5.0" "${MINGW_PACKAGE_PREFIX}-python3<3.6.0"
+  )
+
+  cd ${srcdir}/${_realname}-${pkgver}
+  python3 setup.py install --root=${pkgdir} --prefix=/${MINGW_PREFIX} --optimize=1
```
Comment (Alexpux) [inline mingw-w64-python-traitlets/PKGBUILD]: MINGW_PREFIX contain first slash, so you don't need it here. Second you ned use MSYS2_ARG_CONV_EXCL variable to install files in right locations. Fix it in all your PKGBUILDS.

File: mingw-w64-python-path/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+_realname=path.py
+pkgbase=mingw-w64-python-${_realname}
+pkgname=(
+  "${MINGW_PACKAGE_PREFIX}-python3-path"
+  "${MINGW_PACKAGE_PREFIX}-python2-path"
+)
+pkgver=8.1.2
+pkgrel=1
+pkgdesc="File system based database that uses python pickles"
```
Comment (Alexpux) [inline mingw-w64-python-path/PKGBUILD]: Add ```` (mingw-w64)```` at the end of description in all PKGBUILDs

File: mingw-w64-python-path/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+  cd ${srcdir}/${_realname}-${pkgver}
+  python3 setup.py install --root=${pkgdir} --prefix=/${MINGW_PREFIX} --optimize=1
+  install -d ${pkgdir}/${MINGW_PREFIX}/share/licenses/python3-${_realname}
+  head -n 21 path.py > ${pkgdir}/${MINGW_PREFIX}/share/licenses/python3-${_realname}/LICENSE
+}
+
+package_python2-path() {
+  depends=(
+    "${MINGW_PACKAGE_PREFIX}-python2>=2.7.0" "${MINGW_PACKAGE_PREFIX}-python2<2.8.0"
```
Comment (Alexpux) [inline mingw-w64-python-path/PKGBUILD]: Why need this compare? There are no plans to release python 2.8 at all.

File: mingw-w64-python-path/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+#check() {
+#    cd ${srcdir}/${_realname}-${pkgver}
+#    py.test
+#}
+
+package_python3-path() {
+  depends=(
+    "${MINGW_PACKAGE_PREFIX}-python3>=3.5.0" "${MINGW_PACKAGE_PREFIX}-python3<3.6.0"
```
Comment (Alexpux) [inline mingw-w64-python-path/PKGBUILD]: Will 3.6.0 contain ````path```` module? If no - remove compare

Comment (oroppas) [inline mingw-w64-python-path/PKGBUILD]: python-path module is required by python-pickleshare

Comment (Alexpux) [inline mingw-w64-python-path/PKGBUILD]: Don see relation.

File: mingw-w64-python-traitlets/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  )
+
+  cd ${srcdir}/${_realname}-${pkgver}
+  MSYS2_ARG_CONV_EXCL="--prefix=;--install-scripts=;--install-platlib=" \
+  ${MINGW_PREFIX}/bin/python3 setup.py install --root=${pkgdir} --prefix=${MINGW_PREFIX} --optimize=1
+}
+
+package_python2-traitlets() {
+  depends=(
+    "${MINGW_PACKAGE_PREFIX}-python2>=2.7.0"
```
Comment (Alexpux) [inline mingw-w64-python-traitlets/PKGBUILD]: We not have 2.7 python. Remove comparison

File: mingw-w64-python-simplegeneric/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  cd ${srcdir}/${_realname}-${pkgver}
+  python3 setup.py build
+
+  cd ${srcdir}/${_realname}-${pkgver}-py2
+  python2 setup.py build
+}
+
+package_python3-simplegeneric() {
+  depends=(
+    "${MINGW_PACKAGE_PREFIX}-python3>=3.5.0" "${MINGW_PACKAGE_PREFIX}-python3<3.6.0"
```
Comment (Alexpux) [inline mingw-w64-python-simplegeneric/PKGBUILD]: The same about comparison

File: mingw-w64-python-pickleshare/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  )
+
+  cd ${srcdir}/${_realname}-${pkgver}
+  MSYS2_ARG_CONV_EXCL="--prefix=;--install-scripts=;--install-platlib=" \
+  ${MINGW_PREFIX}/bin/python3 setup.py install --root=${pkgdir} --prefix=${MINGW_PREFIX} --optimize=1
+}
+
+package_python2-pickleshare() {
+  depends=(
+    "${MINGW_PACKAGE_PREFIX}-python2>=2.7.0"
```
Comment (Alexpux) [inline mingw-w64-python-pickleshare/PKGBUILD]: Not have python 2.7

File: mingw-w64-python-pickleshare/PKGBUILD [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+    cd ${srcdir}/${_realname}-${pkgver}
+    nosetests-3.5
+
+    cd ${srcdir}/${_realname}-${pkgver}-py2
+    nosetests-3.5
+}
+
+package_python3-pickleshare() {
+  depends=(
+    "${MINGW_PACKAGE_PREFIX}-python3>=3.5.0" "${MINGW_PACKAGE_PREFIX}-python3<3.6.0"
```
Comment (Alexpux) [inline mingw-w64-python-pickleshare/PKGBUILD]: Comparsion issue

File: mingw-w64-python-path/PKGBUILD [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+  cd ${srcdir}/${_realname}-${pkgver}
+  MSYS2_ARG_CONV_EXCL="--prefix=;--install-scripts=;--install-platlib=" \
+  ${MINGW_PREFIX}/bin/python3 setup.py install --root=${pkgdir} --prefix=${MINGW_PREFIX} --optimize=1
+  install -d ${pkgdir}/${MINGW_PREFIX}/share/licenses/python3-${_realname}
+  head -n 21 path.py > ${pkgdir}/${MINGW_PREFIX}/share/licenses/python3-${_realname}/LICENSE
+}
+
+package_python2-path() {
+  depends=(
+    "${MINGW_PACKAGE_PREFIX}-python2>=2.7.0"
```
Comment (Alexpux) [inline mingw-w64-python-path/PKGBUILD]: not have 2.7

**Pull request merged**